### ISSUE TITLE
Error occurred when converting image

### ISSUE THREAD

**Issue by proxy2008** [dragonflyoss/image-service#24]
使用下面的命令进行镜像转换

/root/Nydus/bin/nydusify convert  、
--containerd-sock /run/docker/containerd/docker-containerd.sock 、
--source-auth  bmV3bGFuZDpOZXdsYW5kQDEyMw==  \
--nydus-image /root/Nydus/bin/nydus-image  \
--source  172.32.150.15:80/nydus/nginx_tyt:1.0.0 \
--target  172.32.150.15:80/paas_public/nginx_tyt:1.0.0-nydus \
--source-insecure true

已经设置的--source-insecure标志为true，但是仍然报了下面的错误
INFO[2020-11-05T18:06:57+08:00] Pulling image 172.32.150.15:80/nydus/nginx_tyt:1.0.0 with platform linux/amd64 
FATA[2020-11-05T18:06:57+08:00] pull source image: pull image: failed to resolve reference "172.32.150.15:80/nydus/nginx_tyt:1.0.0": failed to authorize: failed to fetch oauth token: Post "https://172.32.150.15/service/token": x509: certificate signed by unknown authority 

是这个标志未生效嘛？

**Comment by imeoer** [comment]
Seems the registry token server you are using still be `https` scheme, can you pull the image using `docker pull`?

**Comment by proxy2008** [comment]
> It seems that the token server you're using is still https scheme, can you pull the image using `docker pull`?

yes。

[root@vcapp23 bin]# docker pull 172.32.150.15:80/nydus/nginx_tyt:1.0.0 
1.0.0: Pulling from nydus/nginx_tyt
b7e5ffe1bc2a: Pull complete 
659c02e85607: Pull complete 
db7cc509cc74: Pull complete 
Digest: sha256:67681478d781ad69afed4f6612481e11f346f6b2d5e71ea6e0a3851e40196f89
Status: Downloaded newer image for 172.32.150.15:80/nydus/nginx_tyt:1.0.0
[root@vcapp23 bin]# 


[root@vcapp23 bin]# more /etc/docker/daemon.json 
{
  "insecure-registries":["172.32.150.36","172.32.150.15:80","172.32.150.15"],
  "registry-mirrors": ["https://registry.docker-cn.com", "https://docker.mirrors.ustc.edu.cn"],
  "max-concurrent-downloads": 10,
  "log-driver": "json-file",
  "log-level": "warn",
  "log-opts": {
    "max-size": "10m",
    "max-file": "3"
    },
  "graph": "/home/var/lib/docker"
}
[root@vcapp23 bin]#

**Comment by imeoer** [comment]
@proxy2008 Nydusify will adapt to the auth situation, thanks for the bug report.

**Comment by imeoer** [comment]
@proxy2008 Hi proxy2008, Registry auth in Nydusify has been improved in this PR #25, could you help us to retry? thanks. :)

**Comment by proxy2008** [comment]
I took the code again and compiled it.
After updating the version, a new error occurred when executing the command.

```
[root@vcapp23 bin]# /root/Nydus/bin/nydusify convert \
--nydus-image /root/Nydus/bin/nydus-image \
--source 172.32.150.15:80/nydus/nginx_tyt:1.0.0 \
--target 172.32.150.15:80/paas_public/nginx_tyt:1.0.0-nydus 

FATA[2020-11-27T15:07:34+08:00] fetch source image: Get "https://172.32.150.15:80/v2/": http: server gave HTTP response to HTTPS client 
```

The related configuration remains the same as the original one without any changes.
If you need other useful information, I will try to provide it.

**Comment by imeoer** [comment]
@proxy2008 Sorry for the late reply, we have tried to fix this issue in #32 , could you help us to retry?
Remember to add `--source-insecure` and `--target-insecure` options on Nydusify CLI. :)

**Comment by proxy2008** [comment]
After testing, the new options can be used for image conversion
